src/pages/visualization-view/connector.js

```diff
@@ -13,14 +13,15 @@ const mapStateToProps = state => {
         isSaving: visualizationSaving(state),
         isSaveFailure: visualizationSaveFailure(state),
         isSaveSuccess: visualizationSaveSuccess(state),
-        isSaveable: isVisualizationSaveable(state)
+        isSaveable: isVisualizationSaveable(state),
+        chart: visualizationChart(state)
```

Comment:
Consider using a selector to fetch the chart in the `saveVisualization` saga rather than passing it down to the visualization view component.  This component doesn't otherwise doesn't need to know about this plotly chart data at all.  We're already doing things this way with the query in the `freestyle-query-saga`.

Reply:
ok that makes sense